test/cljs/status_im/test/utils/mixpanel.cljs

```diff
@@ -1,20 +1,41 @@
 (ns status-im.test.utils.mixpanel
-  (:require [cljs.test :refer-macros [deftest is testing async]]
-            [status-im.utils.mixpanel :as mixpanel]))
+  (:require [cljs.test :as t]
+            [status-im.utils.mixpanel :as mixpanel]
+            [cljs.core.async :as async]))
 
-(deftest events
-  (is (not (nil? mixpanel/events))))
+(t/deftest events
+  (t/is (not (nil? mixpanel/events))))
 
-(deftest matches?
-  (is (true? (mixpanel/matches? [:key] [:key])))
-  (is (false? (mixpanel/matches? [:key1] [:key2])))
-  (is (true? (mixpanel/matches? [:key :subkey] [:key])))
-  (is (false? (mixpanel/matches? [:key] [:key :subkey]))))
+(t/deftest matches?
+  (t/is (true? (mixpanel/matches? [:key] [:key])))
+  (t/is (false? (mixpanel/matches? [:key1] [:key2])))
+  (t/is (true? (mixpanel/matches? [:key :subkey] [:key])))
+  (t/is (false? (mixpanel/matches? [:key] [:key :subkey]))))
 
 (def definitions {[:key] {:trigger [:key]} [:key :subkey] {:trigger [:key :subkey]}})
 
-(deftest matching-event
-  (is (empty? (mixpanel/matching-events [:non-existing] definitions)))
-  (is (= 1 (count (mixpanel/matching-events [:key] definitions))))
-  (is (= 2 (count (mixpanel/matching-events [:key :subkey] definitions))))
-  (is (empty? (mixpanel/matching-events [:key1 :another-subkey] definitions))))
+(t/deftest matching-event
```

Comment:
Any reason to introduce `t/`

Reply:
I introduced it because I needed to import "async" and I remembered that the style guide says that I should [avoid `:refer`](https://wiki.status.im/Clojure_Style_Guide#Use_of_:as_and_:refer).

Reply:
@msuess that's actually a good point! In tests we do use `:refer` as it's more idiomatic, we need to update the style guide. Good spot, thanks.

Reply:
Alright, I'll chang it back then :)

Reply:
thanks, sorry for the confusion